### PR TITLE
Add lauren bio page

### PR DESCRIPTION
**Pull request by lauren-avilla**
https://trello.com/c/CBmbuQhu/3-lauren-bio-page

Screen Shot 2022-01-11 at 10 54 32 AM

there is a github logo and linkedin logo that when a user clicks, it directs them to my profiles

### CODE REVIEW

File: src/wintern-bios/2022/TeamLandingPage2022.jsx [hunk 1]
```diff
@@ -1,9 +1,10 @@
 import React, { useState } from 'react';
+import LaurenComponent from './LaurenComponent.jsx';
 
 const TeamLandingPage2022 = () => {
   return (
     <div>
-        OMG. IT'S A BLANK WEBSITE!
+        { <LaurenComponent />}
```
Comment (polarizing) [inline src/wintern-bios/2022/TeamLandingPage2022.jsx:7]: This doesn't need curly braces.

File: src/styles/_lauren-bio.scss [hunk 2]
```diff
@@ -0,0 +1,165 @@
...
+        height: 200px;
+        margin: 0px 10px -250px 100px;
+
+    }
+
+    .linkedin{
+        width: 200px;
+        height: 200px;
+        margin: 0px 10px -250px 400px;
+        //margin top right bottom left
```
Comment (polarizing) [inline src/styles/_lauren-bio.scss:16]: Remove

File: src/styles/_lauren-bio.scss [hunk 3]
```diff
@@ -0,0 +1,165 @@
...
+        color: white;
+        background-color: rgb(93, 102, 179);
+        border-radius:100px;
+        font-size: 15px;
+        padding: 50px;
+        margin: 30px 20px 15px 20px;
+        border-style: dotted;
+    }
+
+    //.aboutmePARA{
```
Comment (polarizing) [inline src/styles/_lauren-bio.scss:80]: Remove comments

**Review by polarizing — APPROVED** [review]
LGTM, pending small code style changes

File: src/styles/_lauren-bio.scss [hunk 4]
```diff
@@ -0,0 +1,165 @@
...
+    }
+
+    .grade{
+        font-family: 'Courier New', Courier, monospace;
+        font-size: to-lower-case($string: "");
+        font-size: medium;
+        color:white;
+        margin: 2px 9px 10px 380px;
+
+
```
Comment (MargaretDiaz4570) [inline src/styles/_lauren-bio.scss:48]: I don't know if we need the extra space between the brackets?

File: src/styles/_lauren-bio.scss [hunk 5]
```diff
@@ -0,0 +1,165 @@
...
+        font-size: to-upper-case($string: "");
+        font-size: 200%;
+        color: white;
+        margin: 270px 19px 10px 400px;
+
+    }
+
+    .grade{
+        font-family: 'Courier New', Courier, monospace;
+        font-size: to-lower-case($string: "");
```
Comment (MargaretDiaz4570) [inline src/styles/_lauren-bio.scss:43]: this looks very cool I don't know you could do that to the font size!

File: src/styles/_lauren-bio.scss [hunk 6]
```diff
@@ -0,0 +1,165 @@
...
+
+    }
+
+    .laurenpic{
+        float: left;
+        margin: 50px 19px 10px 400px;
+        width: 200px;
+        height: 200px;
+        border-radius: 100%;
+        overflow: hidden;
```
Comment (MargaretDiaz4570) [inline src/styles/_lauren-bio.scss:26]: I couldn't figure out how to use overflow so I'm very impressed you did!

File: src/styles/_lauren-bio.scss [hunk 7]
```diff
@@ -0,0 +1,165 @@
...
+        background-color: rgb(93, 102, 179);
+        border-radius:100px;
+        font-size: 15px;
+        padding: 50px;
+        margin: 15px 20px 15px 20px;
+        border-style: dotted;
+
+    }
+
+    //.whyCSPARA{
```
Comment (MargaretDiaz4570) [inline src/styles/_lauren-bio.scss:99]: remove the comments I have to do this for mine as well

File: src/styles/_lauren-bio.scss [hunk 8]
```diff
@@ -0,0 +1,165 @@
...
+    //}
+
+    .hobbies{
+        font-family: 'Courier New', Courier, monospace;
+        color: white;
+        background-color: rgb(93, 102, 179);
+        border-radius:100px;
+        font-size: 15px;
+        padding: 50px;
+        margin: 15px 20px 15px 20px;
```
Comment (MargaretDiaz4570) [inline src/styles/_lauren-bio.scss:133]: is this how you got the text boxes to move? this is very smart I had to do absolute positioning which got very tedious I will try this out next time!

File: src/wintern-bios/2022/LaurenComponent.jsx [hunk 9]
```diff
@@ -0,0 +1,46 @@
...
+        <div className = "laurenbio">
+
+            <a href="https://github.com/lauren-avilla">
+                <img className = "github" src={img2} />
+            </a>
+            
+            <a href = "https://www.linkedin.com/in/lauren-avilla-071467229/">
+                <img className = "linkedin" src={img3} />
+            </a>
+
```
Comment (MargaretDiaz4570) [inline src/wintern-bios/2022/LaurenComponent.jsx:17]: I don't know if we need to add these breaks?

File: src/wintern-bios/2022/LaurenComponent.jsx [hunk 10]
```diff
@@ -0,0 +1,46 @@
...
+            <img className='laurenpic' src={img}/>
+            <br/>
+            <h1 className = "fullname">Lauren Avilla</h1>
+            <h2 className = "grade">Freshman at Hunter College</h2>
+            <h2 className = "major">Major: Computer Science</h2>
+            <h2 className = "location"> Long Island, NY</h2>
+ 
+            <h2 className = "aboutme"><i>ABOUT ME</i><br/><br/>I’m a first generation Filipina American born and raised on Long Island, NY. Ever since moving to Manhattan for college, my love for NYC has grown more and more each day. Three words to describe myself are empathetic, curious, and driven. I recently got into film photography and creative writing! I love having conversations about self-care and mental health!</h2>
```
Comment (MargaretDiaz4570) [inline src/wintern-bios/2022/LaurenComponent.jsx:25]: I like how each class name describes with the section accurately it makes it easy to follow through

File: src/wintern-bios/2022/LaurenComponent.jsx [hunk 11]
```diff
@@ -0,0 +1,46 @@
...
+
+            <h2 className = "hobbies"><i>MY HOBBIES</i><br/><br/>My top three favorite hobbies are yoga, journaling, and thrifting. While I am not the most naturally flexible person out there, yoga challenges me physically and mentally. Everytime I roll out my mat, I feel at peace and grounded. Journaling is my favorite way to reflect on my day and understand my emotions better. I also love thrifting and finding cool pieces for my closet!</h2>
+            <br/>
+
+            <h2 className = "media"><i>MY FAVORITE MEDIA</i><br/><br/>“A True Account of Talking to the Sun at Fire Island” by Frank O’Hara (Poem)<br/>“Small Doses with Amanda Seales” (Podcast)<br/>“The Book of Awakening” by Mark Nepo (Book)</h2>
+            <br/>
+
+        
+
+        </div>
```
Comment (MargaretDiaz4570) [inline src/wintern-bios/2022/LaurenComponent.jsx:42]: your code is very professional looking it looks very efficient!

**Review by MargaretDiaz4570 — APPROVED** [review]
Really like how efficient your code is! Only small thing would be the extra comments and line breaks but that's minor. Really impressive how you managed to figure out how to move the class objects!

File: src/wintern-bios/2022/LaurenComponent.jsx (hunk 11 above)
Comment (MargaretDiaz4570) [reply src/wintern-bios/2022/LaurenComponent.jsx:42]: The end product looks amazing! I love the gradient color background, the presentation is very professional looking and you could definitely use this for jobs! It's very aesthetically pleasing and adding the links was a very cool addition!

**Comment by MargaretDiaz4570** [comment]
I tested it and it looks great!

**Pull request merged**